src/tree.rs

```diff
@@ -243,11 +253,8 @@ impl<'de> Deserialize<'de> for MerkleHashOrchard {
     }
 }
 
-/// Generators for property testing.
-#[cfg(any(test, feature = "test-dependencies"))]
-#[cfg_attr(docsrs, doc(cfg(feature = "test-dependencies")))]
-pub mod testing {
-    #[cfg(test)]
+#[cfg(test)]
+pub mod tests {
```

Comment:
If this is no longer being exposed with a feature, then it shouldn't need to be `pub` anymore. If things in this module are used elsewhere, then make this `pub(crate)`; otherwise make this private.

Reply:
It didn't appear to have anything that needed to be reused; all the proptest generators were in other modules.